docker/README.md

```diff
@@ -0,0 +1,137 @@
+# Docker files for simtools
+
+Docker files for [simtools](https://github.com/gammasim/simtools) development and applications. Prepared packages are available through the [simtools package registry](https://github.com/orgs/gammasim/packages?repo_name=simtools).
+
+[Docker](https://www.docker.com/community-edition#/download) or any compatible software are required to run or build these images.
+
+Types of dockerfiles and containers available:
+
+- [simtools users](#container-for-simtools-users): a container with all software installed (CORSIKA, sim\_telarray, simtools python environment, simtools). Pull latest release with: `docker pull ghcr.io/gammasim/simtools-prod:latest`
+- [simtools developers](#container-for-simtools-developers): a container with CORSIKA, sim\_telarray, and simtools conda environment installed. Pull latest release with: `docker pull ghcr.io/gammasim/simtools-dev:latest`
+- [sim\_telarray](#container-for-simulation-software-corsika--sim_telarray): provides a container with the CORSIKA and sim\_telarray installed. This provides the base image for the previously listed containers. Pull latest release with: `docker pull ghcr.io/gammasim/simtools-simtelarray:latest`
+
+## Container for simtools Users
+
+Provide a container for simtools users, which includes:
+
+- corsika and sim\_telarray
+- packages required by simtools (from pyproject.toml)
+- simtools (main branch)
+
+### Run a simtools-prod container
+
+To run the container in bash
+
+```bash
+docker run ghcr.io/gammasim/simtools-prod:latest bash
+```
+
+In the container, simtools applications are installed and can be called directly (e.g., `simtools-print-array-elements -h`).
+
```

Comment:
It would be interesting to provide a list of all commands (applications) available, or point to the file where we can find the possible commands?

Reply:
This exists: https://gammasim.github.io/simtools/applications.html
(and we have to put some work into the documentation at some point)